cmds/tui.c

```diff
@@ -23,6 +23,7 @@
 #include "utils/dwarf.h"
 
 #define KEY_ESCAPE  27
+#define KEY_SPACE  32
```

Comment:
It'd be better using other name than `KEY_` if it's not used to process keyboard inputs.

Reply:
Changed to `KEY_SPACE -> BLANK`

Reply:
I cannot see the change.

Reply:
I didn't update the change, since there might be additional change with the review.